code/__DEFINES/subsystem-priority.dm

```diff
@@ -26,10 +26,11 @@
 #define SS_INIT_LIGHTING   	0	// Generation of lighting overlays and pre-bake. May cause openturf updates, should initialize before SSzcopy.
 #define SS_INIT_ZCOPY      -1	// Z-mimic flush. Should run after SSoverlay & SSicon_smooth so it copies the smoothed sprites.
 #define SS_INIT_XENOARCH   -2   // Xenoarch is this far below because it can infinite loop if placed in SS_INIT_MISC as it was before, due to some subsystems spawning stuff there.
-#define SS_INIT_HOLOMAP    -3   // Minimap subsystem. Should be initialized after all maps, away sites, ships, planets, etc.
-#define SS_INIT_LOBBY      -4	// Lobby timer starts here. The lobby timer won't actually start going down until the MC starts ticking, so you probably want this last
-#define SS_INIT_PING 	   -5	// Pinger for the clients
-#define SS_INIT_CHAT       -6	// To ensure chat remains smooth during init.
+#define SS_INIT_CODEX      -3   // Codex subsystem. Should be initialized after chemistry and cooking recipes.
```

Comment:
They go into `code\__DEFINES\subsystems.dm` as `INIT_ORDER_*` (as we update them), this is legacy init defines that will disappear as things gets updated